src/containers/MoveListView.js

```diff
@@ -197,14 +219,36 @@ class MoveListView extends React.Component {
 			    	</Row>
 			 	</TabPane>
 
-		  		<TabPane tab="Power" key="4">
+			 	<TabPane tab="Freezes" key="4">
+		  			<Row>
+		  			<Col span={8}>
+			  			<MoveList 
+					    	addMove={this.addMove.bind(this)} 
+					    	deleteMove={this.deleteMove.bind(this)} 
+					    	moves_list={this.state.moves_list} 
+					    	select_move={this.select_move.bind(this)}
+					    	current_tab={this.state.current_tab}
+				    	/>
+				    </Col>
+				   	<Col span={16}>
+					   	<MoveDetail 
```

Comment:
Can you look into issue https://github.com/Helen300/BboyGen/issues/20. If it's too difficult we can do it in another pull request.